infrastructure/templates/datafactory/components/template.json

```diff
@@ -97,7 +109,7 @@
                 "annotations": [],
                 "type": "AzureSqlDatabase",
                 "typeProperties": {
-                    "connectionString": "[parameters('ls_sql_statistics_connectionString')]"
+                    "connectionString": "[concat('Data Source=tcp:', reference(concat('Microsoft.Sql/servers/', variables('sqlServerName'))).fullyQualifiedDomainName, ',1433;Initial Catalog=', 'statistics', ';User Id=', variables('sql-user'), '@', reference(concat('Microsoft.Sql/servers/', variables('sqlServerName'))).fullyQualifiedDomainName, ';Password=', variables('sql-admin-password'), ';')]"
```

Comment:
I'm not sure this actually fully solves the vault problem as this will still write a plain text value. Though i'm happy for this to go in.

Im not 100% sure but i think there might be a way to reference vault directly in datafactory for the full connection string.

Reply:
yes I can do that if it's OK to put the full connectionstring in?

Reply:
ok have now added vault to be used as linked datafactory service & added datafactory user to both dbs and vault secrets for connection strings @jamescutts